<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MembershipEdit</class>
 <widget class="QWidget" name="MembershipEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="mandate_GroupBox">
       <property name="title">
        <string>Mandattid</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="wholeyear_radioButton">
          <property name="text">
           <string>Hela året</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="laborday_radioButton">
          <property name="text">
           <string>Vapp till vapp</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="otherMandate_radioButton">
          <property name="text">
           <string>Annan tid</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="startAndEndTimeWidget" native="true">
       <layout class="QFormLayout" name="startEndLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="startdatumLabel_2">
          <property name="text">
           <string>Startdatum</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="startTime_fld"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="slutdatumLabel_2">
          <property name="text">
           <string>Slutdatum</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="endTime_fld"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="startYearWidget" native="true">
       <layout class="QFormLayout" name="startYearLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="frNRLabel">
          <property name="text">
           <string>Från år</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="startYear_spinBox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
